pkg/controller/performanceprofile/components/machineconfig/machineconfig_test.go

```diff
@@ -24,6 +24,7 @@ const expectedSystemdUnits = `
 
           [Service]
           Environment=NON_ISOLATED_CPUS=2-3
+          Environment=NON_ISOLATED_CPU_MASK_INVERT=3ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff3
```

Comment:
@vladikr  this is the format we got from the conversion functions (for the list 2-3)  does it look ok from your side ? or do we need to add a delimiter here?

Reply:
It looks fine, I think.

When checking myself against an online calculator, based on 64 CPUs [1] - I see the same result from our function when running it on 64 CPUs:

reserved cpus mask:  0c
inverted cpus mask:  fffffffffffffff3

and then the following, when running on 254 CPUs
reserved cpus mask:  0c
inverted cpus mask:  3ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff3



> or do we need to add a delimiter here?

I'm not sure.. which delimiter?

Reply:
by the documentation:
"export IRQBALANCE_BANNED_CPUS=ff000000,00000001' would prevent irqbalance from assigning irqs to the 1st (cpu0) and 57th-64th cpus (cpu56-cpu63)"
Im not sure its critical to add the comma though.

Reply:
hmm, right, but that's something completely different. 

I mean

ff00000000000000 - is a mask (not inverted) of cpus 56-63
and 00000001 is a mask of cpu 0

while 3ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff3 is an inverted mask

so AFAIU, gets a list of cpu masks (not invereted)

Reply:
right! :) , need to make sure it will not be an issue for less than 64-256 cpu's, but i think its ok.